test/new-e2e/tests/agent-platform/package-signing/package_signing_test.go

```diff
@@ -86,7 +85,6 @@ func TestPackageSigningComponent(t *testing.T) {
 			&packageSigningTestSuite{osName: platform},
 			e2e.WithProvisioner(awshost.ProvisionerNoFakeIntake(
 				awshost.WithEC2InstanceOptions(ec2.WithAMI(ami, osDesc, osDesc.Architecture)),
-				awshost.WithAgentOptions(agentparams.WithPipeline(os.Getenv("CI_PIPELINE_ID"))),
 			)),
 			//e2e.WithAgentParams(agentparams.WithPipeline(os.Getenv("CI_PIPELINE_ID"), "x86_64")), e2e.WithVMParams(ec2params.WithOS(testedOS))),
 			e2e.WithStackName(fmt.Sprintf("pkgSigning-%s-%s", platform, os.Getenv("CI_PIPELINE_ID"))),
```

Comment:
I think we should keep `CI_PIPELINE_ID` for stack name. `E2E_PIPELINE_ID` refers to pipeline we target when choosing the version of the agent we want to install

Reply:
I also think that these `WithStackName` are not really needed since the framework itself will create a unique ID with the CI job id. Let me clean this up